bin/start_sync_stop.py

```diff
@@ -0,0 +1,137 @@
+#!/usr/bin/env python
+"""A script to test rippled in an infinite loop of start-sync-stop.
+
+- Expects 'rippled' to be on the `PATH` and 'rippled.cfg' to be in the current
+  directory.
```

Comment:
I keep many exe and config files around. It would be nice if the executable and config file could be set with command line arguments.